src/util/image/MapPropertiesMaker.java

```diff
@@ -212,8 +212,7 @@ public void focusLost(final FocusEvent e) {
         GridBagConstraints.NONE, new Insets(10, 10, 10, 10), 0, 0));
     final JButton showMore = new JButton("Show All Options");
     showMore.addActionListener(SwingAction.of("Show All Options", e -> {
-      @SuppressWarnings("rawtypes")
-      final Tuple<PropertiesUI, List<MapPropertyWrapper>> propertyWrapperUI =
+      final Tuple<PropertiesUI, List<MapPropertyWrapper<Object>>> propertyWrapperUI =
```

Comment:
Same, we should have a TODO here to fix this properly. The thing we want to  avoid is casting, and having a type 'Object' means we'll be doing some casting when we read this data back.